### PR TITLE
added beat in contact forms

### PR DESCRIPTION
**Pull request by pareta1107**

### CODE REVIEW

File: scripts/privyr-generic-cf-integration.js [hunk 1]
```diff
@@ -66,6 +70,17 @@ export default class PrivyrGenericCfIntegration {
         }
     };
 
+    _sending_beat() {
+        let payload = {
+            "license_code": this.license_code,
+            "full_url": window.location.href,
```
Comment (stripathi669) [inline scripts/privyr-generic-cf-integration.js:76]: :point_up:  We may not need the GET param right? For ex:
Same form on `www.privyr.com?gclid=232bfjhbf2` and `www.privyr.com?gclid=asafqw` should be same heartbeat. 

What do you think?

File: scripts/privyr-generic-cf-integration.js [hunk 2]
```diff
@@ -28,6 +28,16 @@ export default class PrivyrGenericCfIntegration {
         return luid;
     }
 
+    _post_xhr_request(post_url, payload, async=false) {
```
Comment (stripathi669) [inline scripts/privyr-generic-cf-integration.js:31]: Will this get compiled to ECMA 5? Because it doesn't support async=Flase in signature

**Review by stripathi669 — CHANGES_REQUESTED** [review]
Added a couple of comments.

File: scripts/privyr-generic-cf-integration.js (hunk 2 above)
Comment (pareta1107) [reply scripts/privyr-generic-cf-integration.js:31]: Yes.. It will.

**Pull request merged**